common/remote/sockfile.cpp

```diff
@@ -1922,9 +1922,15 @@ class CRemoteFile : public CRemoteBase, implements IFile
 
     void setCreateFlags(unsigned cflags)
     {
+        flags &= 0xffff;
```

Comment:
See other comments re. serializes/deserializing as unsigned short.

As other code forces bottom 2 bytes of setCreateFlags value to be used, setCreateFlags should be change to:
```
setCreateFlags(unsigned short cflags)
```

worth double checking not any impact elsewhere, but shouldn't be.